src/point_one/polaris/polaris_client.cc

```diff
@@ -224,7 +260,7 @@ void PolarisClient::Run(double timeout_sec) {
         running_ = false;
         break;
       } else if (ret != POLARIS_SUCCESS) {
-        LOG(WARNING) << "Authentication failed. Retrying.";
+        LOG(WARNING) << "Authentication failed. (" << ret << ") Retrying.";
```

Comment:
"Authentication failed. (3) Retrying." looks a little odd. Maybe change to "...Retrying. [error=3]"?